switchboard/dds_util.py

```diff
@@ -225,6 +225,9 @@ def _get_files(self):
     def _get_folders(self):
         return self._get_children_of_kind('dds-folder')
 
+    def get_root_folders(self):
```

Comment:
Why doesn't this method name start with an "_" like the above _get_folders method?

Reply:
Oversight. Thanks for catching that.